### ISSUE TITLE
The button that closes the picker reaches the game; focus loss doesn't pause

### ISSUE THREAD

**Issue by starquake** [zx-sidekick/zx-sidekick#167]
> 🤖 **Issue by Claude** (AI pair-programmer working with @starquake) — posted through @starquake's account.

Found in the code review of 2026-09-24, in both games' windows. Three reviewers found the first point independently.

## What was wrong

1. **The pad button that closes the picker also reached the game.** Each game's `hold_for_picker` returned an empty pad for one frame only. On the next frame the confirm button, still held, counted as pressed:
   - in Starquake, keeping settings with A made Blob lay a platform (A is down);
   - in Manic Miner, Willy jumped;
   - if the picker had been opened from a pause, the freeze saw a fresh press and the game started again by itself.
2. **The picker's keys were written out twice.** Esc, the arrows and Enter, and the pad's Select or B, the d-pad and A, were mapped to the same five calls in each game.
3. **Switching away from the window didn't pause the game.** The held keys were let go, but play went on, so Cmd-Tab away mid-room and Blob or Willy died unattended.

## Plan

- [x] `Gamepad::hold_back_held`: from the next poll, keep what is still held, joystick bits and Start, from the game until each is let go. Both games call it as the picker closes. It has a test.
- [x] `sidekick_frontend::picker`: a `Picker` trait and `key` and `pad` drivers, implemented by Starquake's `Guidance` and Manic Miner's `Picker`. It has a test that the keys and the pad do the same things.
- [x] `Shared::unfocused`: set by the window when it loses the focus, and taken by each game's loop as a pause, through the same freeze the pause key uses.

## Open questions

Answer in the PR or here:

```
# keep your pick, delete the rest
Q1 focus loss = pauses a game in play, like the pause key (rec) | does nothing, as before
notes =
```

**Comment by starquake** [comment]
> 🤖 **Comment by Claude** (AI pair-programmer working with @starquake) — posted through @starquake's account.

> 🤖 **Next steps**

Filed and built at @starquake's word in chat on 2026-09-24: "Do a code review of the entire codebase … For every issue found create a ticket and a PR of the recommended fix so I can review and merge. Do not ask any questions, put them in the ticket." The fix is #168, with the full gate passing. The card is in `Your review`.

Next: review the PR and add `ready to merge`; any question left open is in the body above.
